### PR TITLE
Add some instruction test

### PR DESCRIPTION
**Pull request by musenming**
unit test (test_instructions)

### CODE REVIEW

**Comment by musenming** [comment]
Please don't merge, still working on this PR.

File: fengyong/instructions.py [hunk 1]
```diff
@@ -177,7 +177,7 @@ def slti_(rt, rs, imm):
 #    Registers.reg_set(rt, RegData(Registers.reg_get(rs) < RegData(imm)))
 
 # Shift right arithmetic
-# def sra_(rd, rt, shamt)
+# def sra_(rd, rt, shamt)f
```
Comment (loopyme) [inline fengyong/instructions.py:180]: please remove this

**Review by loopyme — CHANGES_REQUESTED** [review]

**Comment by loopyme** [comment]
Hi, @musenming .To let us pull your changes, please solve the conflicts and merge [this PR](https://github.com/musenming/pymips/pull/1) to active Travis-CI in you fork.

**Comment by loopyme** [comment]
Move to #17

**Pull request closed without merging**